core/src/test/scala/integration/kafka/api/ProducerIdExpirationTest.scala

```diff
@@ -183,7 +183,11 @@ class ProducerIdExpirationTest extends KafkaServerTestHarness {
     )
 
     // Update the expiration time to a low value again.
-    admin.incrementalAlterConfigs(producerIdExpirationConfig("100"))
+    admin.incrementalAlterConfigs(producerIdExpirationConfig("100")).all().get()
```

Comment:
Does the all + get ensure the call actually completes?

Reply:
Yes